assembl/static2/js/app/client.js

```diff
@@ -1,10 +1,13 @@
 import { ApolloClient, toIdValue, IntrospectionFragmentMatcher } from 'react-apollo';
 import { createNetworkInterface } from 'apollo-upload-client';
 import * as Sentry from '@sentry/browser';
+import Cookies from 'js-cookie';
+import co from 'co';
```

Comment:
You can't use async / await inside a regular callback - it has to be itself wrapped inside something that is async - this block of code is a middleware `applyMiddleware: function (req, next) {`so you cannot yield or async / await, and so it's important to wait for the token before advancing otherwise javascript just keeps on processing. Try for yourself, you can't add an async - you can possible add a Promise but this is cleaner IMHO

Reply:
Okay fair. Se queda.